src/components/ResizeList/ResizeList.tsx

```diff
@@ -0,0 +1,98 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import '../../yamui';
+import * as React from 'react';
+import { join } from '../../util/classNames';
+import { BaseComponentProps } from '../../util/BaseComponent/props';
+import { ResizeGroup } from 'office-ui-fabric-react/lib/ResizeGroup';
```

Comment:
Ha, I didn't know this existed. Do they rely on viewport `resize` events, or do they include a `ResizeObserver` polyfill?

Reply:
they're literally listening to a throttled window resize event
https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/ResizeGroup/ResizeGroup.base.tsx#L342